src/valkeymodule.h

```diff
@@ -1649,6 +1691,19 @@ VALKEYMODULE_API int (*ValkeyModule_RdbSave)(ValkeyModuleCtx *ctx,
                                              ValkeyModuleRdbStream *stream,
                                              int flags) VALKEYMODULE_ATTR;
 
+VALKEYMODULE_API int (*ValkeyModule_RegisterScriptingEngine)(ValkeyModuleCtx *ctx,
+                                                             const char *engine_name,
+                                                             void *engine_ctx,
```

Comment:
There's a type for engine context. Is this one of those?

```suggestion
                                                             ValkeyModuleScriptingEngineCtx *engine_ctx,
```